src/test/peerman_tests.cpp

```diff
@@ -0,0 +1,188 @@
+// Copyright (c) 2011-2021 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <arith_uint256.h>
+#include <banman.h>
+#include <chainparams.h>
+#include <net.h>
+#include <net_processing.h>
+#include <netmessagemaker.h>
+#include <pubkey.h>
+#include <script/sign.h>
+#include <script/signingprovider.h>
+#include <script/standard.h>
```

Comment:
nit: Seems like a lot of these includes are unnecessary and can be removed; e.g. I don't see any arith_uint256-, pubkey- or script-related stuff used in the unit test. With the following diff it still compiles:
```diff
diff --git a/src/test/peerman_tests.cpp b/src/test/peerman_tests.cpp
index 8ab87be6b..9d0c02fd0 100644
--- a/src/test/peerman_tests.cpp
+++ b/src/test/peerman_tests.cpp
@@ -2,16 +2,11 @@
 // Distributed under the MIT software license, see the accompanying
 // file COPYING or http://www.opensource.org/licenses/mit-license.php.

-#include <arith_uint256.h>
 #include <banman.h>
 #include <chainparams.h>
 #include <net.h>
 #include <net_processing.h>
 #include <netmessagemaker.h>
-#include <pubkey.h>
-#include <script/sign.h>
-#include <script/signingprovider.h>
-#include <script/standard.h>
 #include <serialize.h>
 #include <span.h>
 #include <test/util/net.h>
@@ -22,7 +17,6 @@
 #include <timedata.h>
 #include <validation.h>

-#include <array>
 #include <optional>
 #include <stdint.h>

```